INN/umbrella-workdayminn
# Styling for footer logos

## Changes

This pull request makes the following changes:

- Adds styling for the footer logos using the `TC Logo Slider` plugin

Desktop:
![Screen Shot 2019-08-07 at 4 24 09 PM](https://user-images.githubusercontent.com/18353636/62655565-4513b900-b930-11e9-91f0-bc21fe53b85f.png)

Mobile:
![Screen Shot 2019-08-07 at 4 24 24 PM](https://user-images.githubusercontent.com/18353636/62655564-4513b900-b930-11e9-988a-005ae232be49.png)

## Why



For #17 

## Testing/Questions

Features that this PR affects:

- Footer logos


Questions that need to be answered before merging:

- [ ] Does it look and function as expected?

Steps to test this PR:

1. Install `TC Logo Slider` plugin https://wordpress.org/plugins/tc-logo-slider/
2. Add in 4 logos and assign them all to new `footer logos` category
3. Add text widget to Footer 1 widget area and paste in `[tc-logo-slider tcls-grid-style="one" logo_cat="footer-logos"]`
4. Logo settings:
   - Auto play -> no
   - Carousel loop -> no
   - Item in Tablets -> 4
   - Item in Mobile -> 4
   Advanced:
   - Show Navigation -> no
   - Show Dots -> no
   - Crop logo image -> no
